install/kubernetes/cilium/charts/hubble/values.yaml

```diff
@@ -0,0 +1,99 @@
+# image used for hubble server
+image:
+  # repository of the docker image
+  repository: quay.io/cilium/hubble
+  # tag is the container image tag to use
+  tag: latest
+  # pullPolicy is the container image pull policy
+  pullPolicy: Always
+
+# URL to listen to client requests. If this parameter is not specified, it
+# defaults to using unix domain socket.
+listenClientUrls: ~
+
+# groupName specifies the group to use for unix domain socket Hubble uses for
+# the gRPC server. Defaults to "hubble".
+groupName: ~
+
+# Server URL to connect to hubble server. If this parameter is not specified,
+# it defaults to using unix domain socket.
+server: ~
+
+# Specify the default server for observe and status commands when the --server
+# option is not specified. It defaults to "unix:///var/run/hubble.sock".
+defaultServer: ~
+
+# maxFlows the server will store in memory
+maxFlows: ~
+
+# Namespace in which Cilium is installed
+ciliumNamespace: ~
+
+# Configuration for container runtime interface. When enabled, Hubble uses CRI to get additional
+# metadata such as container name and image name.
+cri:
+  # CRI is disabled if runtime field is set to "". Set this to one of the following values to
+  # enable it.
+  #   - docker
+  runtime: ""
+
+metrics:
+  enabled:
+   - drop
+  address: ":6943"
+  # Create a prometheus-operator servicemonitor
+  serviceMonitor:
+    enabled: false
+
+# Configuration for hubble ui
+ui:
+  enabled: false
+  image:
+    # repository of the docker image
+    repository: quay.io/cilium/hubble-ui
+    # tag is the container image tag to use
+    tag: latest
+    # pullPolicy is the container image pull policy
+    pullPolicy: Always
+  clusterDomain: cluster.local
+  replicas: 1
+  # Resource requests and limits for the 'hubble-ui' container of the 'hubble-ui' deployment, such as
+  #
+  #     resources:
+  #       limits:
+  #         cpu: 1000m
+  #         memory: 1024Mi
+  #       requests:
+  #         cpu: 100m
+  #         memory: 64Mi
+  resources: {}
+  # The priority class system-node-critical marks add-on pods as critical to the node itself.
+  # This priority class is only valid under the kube-system namespace.
+  priorityClassName: ""
+
+ingress:
+  enabled: false
+  annotations: {}
+    # kubernetes.io/ingress.class: nginx
+    # kubernetes.io/tls-acme: "true"
+  path: /
+  hosts:
+    - chart-example.local
+  tls: []
+  #  - secretName: chart-example-tls
+  #    hosts:
+  #      - chart-example.local
+
+# Resource requests and limits for the 'hubble' container of the 'hubble' daemonset, such as
+#
+#     resources:
+#       limits:
+#         cpu: 1000m
+#         memory: 1024Mi
+#       requests:
+#         cpu: 100m
+#         memory: 64Mi
+resources: {}
+# The priority class system-node-critical marks add-on pods as critical to the node itself.
+# This priority class is only valid under the kube-system namespace.
+priorityClassName: system-node-critical
```

Comment:
please add the constraint to specify the `priorityClassName` based on the k8s version

Reply:
Can you elaborate on this. I am not sure if I am getting it right.

Reply:
I think I added similar to the one from the cilium/daemonset.yaml, to add constraints for k8s version.
Please let me know if this would work.

Reply:
yes, that exactly.

Reply:
@soumynathan this does not seem to be addressed

Reply:
I will fix this, I am working on other values to be realigned.